docs/source/introduction/quickstart.ipynb

```diff
@@ -22,45 +22,56 @@
     "\n",
```

Comment:
<ul><li>over the solution rather than simply a point estimate.</li><li>particular case</li><li>`Normal' vs 'Normal' or "Normal" (markdown doesnt support latex style quotation marks)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>done</p>